### PR TITLE
Example/responsive card

### PR DESCRIPTION
**Pull request by Selman-S**
example finished

### CODE REVIEW

File: libs/ui/src/examples/ResponsiveCard/ResponsiveCard.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      p={10}
+      gap={8}
+      borderRadius="8px"
+      flexDirection={['column', null, 'row']}
+      m={5}
+    >
+      <Stack>
+        <Image
+          w={['300px', null, 'full']}
+          h={['300px', null]}
```
Comment (7alip) [inline libs/ui/src/examples/ResponsiveCard/ResponsiveCard.tsx:19]:
```suggestion
          h={['300px', null, 'full']}
```

Wouldn't it be better if the image always fills the space vertically?

File: libs/ui/src/examples/ResponsiveCard/ResponsiveCard.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          src={randomPic}
+          borderRadius={['full', null, 'none']}
+          m="auto"
+        />
+      </Stack>
+
+      <Stack spacing={2} w={['full', null, '80%']}>
```
Comment (7alip) [inline libs/ui/src/examples/ResponsiveCard/ResponsiveCard.tsx:26]: Is 80% necessary here?

**Review by 7alip — APPROVED** [review]
Thank you, it looks nice.

File: libs/ui/src/examples/ResponsiveCard/ResponsiveCard.tsx (hunk 2 above)
Comment (Selman-S) [reply libs/ui/src/examples/ResponsiveCard/ResponsiveCard.tsx:26]: It isn't necessary . I will update

**Comment by 7alip** [comment]
Thank you, it looks perfect. Do you want to merge it?

**Pull request merged**